spec/project_spec.rb

```diff
@@ -5,12 +5,12 @@
     let(:cop_names) do
       Dir.glob(File.join(File.dirname(__FILE__), '..', 'lib', 'rubocop', 'cop',
                          'rspec', '*.rb'))
-        .map do |file|
-          cop_name = File.basename(file, '.rb')
-            .gsub(/(^|_)(.)/) { Regexp.last_match(2).upcase }
+         .map do |file|
+        cop_name = File.basename(file, '.rb')
```

Comment:
Yup, forgot to update `.rubocop.yml` with the new `MutilineMethodCallIndentation` cop.